.github/scripts/download-node-tooling.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
```

Comment:
Great tip! I went ahead and added `set -euo pipefail`.

Reply:
I prefer just `#!/bin/bash -e`. What's wrong with that?

Reply:
Shebang lines can only accept one argument (a kernel limitation), so a separate call to `set` is required to get the `-o pipefail` part, which is essential, since it makes bash more predictable but doesn't mess with coding style.

`-u` mode could be added as part of a shebang line (`#!/bin/bash -eu`), and I also consider it important since bad variable refs are a frequent source of silent bugs in bash scripts. However using it does require changing your coding style a bit. Worth it in my opinion.

Reply:
I went ahead and removed this, I don't think it is worth the hassle right now. This is a rather simple script that is not mission critical. Also, all the existing scripts we have either set nothing at all, or do so inconsistently, so I'd say if we are going to have this discussion we should open a new issue and apply whatever convention comes out of it to all scripts.